keyboards/iron180/keymaps/default/keymap.c

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2012,2013 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#define ____ KC_NO
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

Reply:
Actually, I'm seeing this a lot, why is this here?  Did you use a site to generate the code?  If so, which?  

Or is it copying from another keyboard? 

Or?

Reply:
I had no idea that there already was a quantum definition, hence why I was using that four underscores one. I've always done that, can't remember if it was an old habit or just copied over. I will try to use the seven underscores one from now on

Reply:
Commited in 9f6d2dbea